ee/codegen/src/project.ts

```diff
@@ -974,6 +985,10 @@ ${errors.slice(0, 3).map((err) => {
       return await node.persist();
     });
 
+    if (this.workflowContext.skipInitFiles) {
+      return [...nodePromises];
+    }
```

Comment:
[blocking] Let's remove this flag entirely, and just skip generating the node files when we're ready

We may want to do the codegen piece in its own PR, bc I think I thought of a gnarly edge case related to the display file's init.py that I'll tee up once the loader changes go in